dependencies/pip/requirements.in

```diff
@@ -72,6 +72,7 @@ unicodecsv
 uWSGI
 xlrd
 xlwt
```

Comment:
Would we like to replace `xlwt` everywhere?

Reply:
Maybe. I left xlwt in `sync_kobocat_xforms` because I thought that maybe xlwt is faster than XlsxWriter, and that XLS files are generally smaller than XLSX—but I haven't tested my assumptions.

Probably what would be better is to convert directly from CSV to KPI asset JSON instead of using XLS(X) at all:
https://github.com/kobotoolbox/kpi/blob/5dcdce30d299e239934a812ec3675394de019b63/kpi/management/commands/sync_kobocat_xforms.py#L169-L175

Reply:
Indeed. 
Let's keep this for later then.